Description: Unfix clap version
 which was set in upstream commit ae089411de, being "the highest clap version
 that is compatible with the current MSRV". OTOH, parsec-service is currently
 only in testing, which has rustc 1.80, so that does not really matter.
Forwarded: not-needed
Last-Update: 2024-10-13
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -52,3 +52,3 @@
 [dependencies.clap]
-version = "=4.3.24"
+version = "4.3.24"
 features = [
